src/video_core/renderer_vulkan/vk_sampler_cache.cpp

```diff
@@ -23,7 +23,16 @@ static std::optional<vk::BorderColor> TryConvertBorderColor(std::array<float, 4>
     } else if (color == std::array<float, 4>{1, 1, 1, 1}) {
         return vk::BorderColor::eFloatOpaqueWhite;
     } else {
-        return {};
+        if (color[0] + color[1] + color[2] > 1.35f) {
```

Comment:
This can be unindented as follows:

```suggestion
        if (color[0] + color[1] + color[2] > 1.35f) {
            // If color elements are brighter than roughly 0.5 average, use white border
            return vk::BorderColor::eFloatOpaqueWhite;
        }
        if (color[3] > 0.5f) {
            return vk::BorderColor::eFloatOpaqueBlack;
        }
        return vk::BorderColor::eFloatTransparentBlack;
```

Reply:
yeah, right :p